### ISSUE TITLE
[bug] L1->L2 deposit fails due to t["l1WethBridge"] being None

### ISSUE THREAD

**Issue by sanekmelnikov** [zksync-sdk/zksync2-python#74]
With the introduction of new chain deployments that feature shared bridges, I believe the concepts of 'l1WethBridge' and 'l2WethBridge' are now obsolete.
When attempting to make a deposit from L1 to L2, I encounter an error because `None` is being passed to `to_checksum_address`, resulting in issues with [these lines](https://github.com/zksync-sdk/zksync2-python/blob/master/zksync2/module/zksync_module.py#L187-L188)

**Comment by petarTxFusion** [comment]
Could you please share which version of the SDK are you using and which network. I have tried with latest version with mainnet, testnet and local and it returns `'0x0000000000000000000000000000000000000000'` as it should since support for `weth` is dropped

**Comment by sanekmelnikov** [comment]
I can confirm that both the official mainnet and the Sepolia testnet are functioning properly for me as well.
The issue seems to be with locally deployed zkSync chains which I can't share since it's local.
It was created with [zk_inception](https://github.com/matter-labs/zksync-era/tree/main/zk_toolbox).

I replaced:
`t["l1WethBridge"]` -> `t.get("l1WethBridge") or ZERO_ADDRESS`
`t["l2WethBridge"]` -> `t.get("l2WethBridge") or ZERO_ADDRESS`
and it worked for me.

Feel free to close the issue since I was able to resolve it locally, just wanted to bring awareness.

I am using zksync2@1.2.0
